lib/glue/tracker.rb

```diff
@@ -33,6 +33,18 @@ def report finding
     @findings << finding
   end
 
+  def get_worst_finding
+    worst = nil
+    @findings.each do |finding|
```

Comment:
Possible to clarify? Perhaps something like:

    @findings.map { |f| f.severity }.max

Would simplify this?